testing fails with pyramid 1.6

Here's an excerpt: 

[   16s] FAIL: test_it (pyramid_zcml.tests.test_units.TestAssetDirective)
[   16s] ----------------------------------------------------------------------
[   16s] Traceback (most recent call last):
[   16s]   File "/home/abuild/rpmbuild/BUILD/pyramid_zcml-1.0.0/pyramid_zcml/tests/test_units.py", line 653, in test_it
[   16s]     'fixtureapp/')])
[   16s] AssertionError: Lists differ: [(<module 'pyramid_zcml.tests'... != [(<module 'pyramid_zcml.tests'...

Also, PyPI points to non existing doc. BTW, PyPI points to version 1.0.0, while this repo still has 0.9.2 as the latest release...

Would be nice to get those PR merged We're facing the non working test during packaging @openSUSE
```log
[   23s] Executing(%check): /bin/sh -e /var/tmp/rpm-tmp.PCz2gH
[   23s] + umask 022
[   23s] + cd /home/abuild/rpmbuild/BUILD
[   23s] + cd pyramid_zcml-1.0.0
[   23s] + python3 setup.py nosetests --with-coverage
[   23s] running nosetests
[   23s] running egg_info
[   23s] writing requirements to pyramid_zcml.egg-info/requires.txt
[   23s] writing dependency_links to pyramid_zcml.egg-info/dependency_links.txt
[   23s] writing entry points to pyramid_zcml.egg-info/entry_points.txt
[   23s] writing top-level names to pyramid_zcml.egg-info/top_level.txt
[   23s] writing pyramid_zcml.egg-info/PKG-INFO
[   23s] reading manifest file 'pyramid_zcml.egg-info/SOURCES.txt'
[   23s] writing manifest file 'pyramid_zcml.egg-info/SOURCES.txt'
[   23s] /usr/lib/python3.4/site-packages/nose/config.py:435: DeprecationWarning: Use of multiple -w arguments is deprecated and support may be removed in a future release. You can get the same behavior by passing directories without the -w argument on the command line, or by using the --tests argument in a configuration file.
[   23s]   DeprecationWarning)
[   24s] ..........................................F................................................................
[   24s] ======================================================================
[   24s] FAIL: test_it (pyramid_zcml.tests.test_units.TestAssetDirective)
[   24s] ----------------------------------------------------------------------
[   24s] Traceback (most recent call last):
[   24s]   File "/home/abuild/rpmbuild/BUILD/pyramid_zcml-1.0.0/pyramid_zcml/tests/test_units.py", line 653, in test_it
[   24s]     'fixtureapp/')])
[   24s] AssertionError: Lists differ: [(<mo[122 chars]/', <pyramid.config.assets.PackageAssetSource [22 chars]00>)] != [(<mo[122 chars]/', <module 'pyramid_zcml.tests' from '/home/a[84 chars]p/')]
[   24s] 
[   24s] First differing element 0:
[   24s] (<module 'pyramid_zcml.tests' from '/home/abuild/rpmbuild/BUILD/pyramid_zcml-1.0.0/pyramid_zcml/tests/__init__.py'>, 'fixtures/', <pyramid.config.assets.PackageAssetSource object at 0x7f87c4b63b00>)
[   24s] (<module 'pyramid_zcml.tests' from '/home/abuild/rpmbuild/BUILD/pyramid_zcml-1.0.0/pyramid_zcml/tests/__init__.py'>, 'fixtures/', <module 'pyramid_zcml.tests' from '/home/abuild/rpmbuild/BUILD/pyramid_zcml-1.0.0/pyramid_zcml/tests/__init__.py'>, 'fixtureapp/')
[   24s] 
[   24s]   [(<module 'pyramid_zcml.tests' from '/home/abuild/rpmbuild/BUILD/pyramid_zcml-1.0.0/pyramid_zcml/tests/__init__.py'>,
[   24s]     'fixtures/',
[   24s] -   <pyramid.config.assets.PackageAssetSource object at 0x7f87c4b63b00>)]
[   24s] +   <module 'pyramid_zcml.tests' from '/home/abuild/rpmbuild/BUILD/pyramid_zcml-1.0.0/pyramid_zcml/tests/__init__.py'>,
[   24s] +   'fixtureapp/')]
[   24s]
[   24s] Name                                                 Stmts   Miss  Cover   Missing
[   24s] ----------------------------------------------------------------------------------
[   24s] pyramid_zcml.py                                        331      0   100%
[   24s] pyramid_zcml/_compat.py                                  2      0   100%
[   24s] pyramid_zcml/scaffolds.py                                5      0   100%
[   24s] pyramid_zcml/tests.py                                    0      0   100%
[   24s] pyramid_zcml/tests/ccbugapp.py                           0      0   100%
[   24s] pyramid_zcml/tests/ccbugapp/views.py                     5      0   100%
[   24s] pyramid_zcml/tests/defpermbugapp.py                      6      0   100%
[   24s] pyramid_zcml/tests/exceptionviewapp.py                   0      0   100%
[   24s] pyramid_zcml/tests/exceptionviewapp/models.py           12      0   100%
[   24s] pyramid_zcml/tests/exceptionviewapp/views.py            12      0   100%
[   24s] pyramid_zcml/tests/fixtureapp.py                         0      0   100%
[   24s] pyramid_zcml/tests/fixtureapp/models.py                  4      0   100%
[   24s] pyramid_zcml/tests/fixtureapp/subpackage.py              0      0   100%
[   24s] pyramid_zcml/tests/fixtureapp/views.py                   9      0   100%
[   24s] pyramid_zcml/tests/hybridapp.py                          0      0   100%
[   24s] pyramid_zcml/tests/hybridapp/views.py                   19      0   100%
[   24s] pyramid_zcml/tests/includeoverrideapp.py                 0      0   100%
[   24s] pyramid_zcml/tests/includeoverrideapp/sub.py             0      0   100%
[   24s] pyramid_zcml/tests/includeoverrideapp/sub/views.py       2      0   100%
[   24s] pyramid_zcml/tests/includeoverrideapp/views.py           2      0   100%
[   24s] pyramid_zcml/tests/localeapp.py                          0      0   100%
[   24s] pyramid_zcml/tests/permbugapp.py                         6      0   100%
[   24s] pyramid_zcml/tests/restbugapp.py                         0      0   100%
[   24s] pyramid_zcml/tests/restbugapp/views.py                  10      0   100%
[   24s] pyramid_zcml/tests/routesapp.py                          0      0   100%
[   24s] pyramid_zcml/tests/routesapp/views.py                    4      0   100%
[   24s] pyramid_zcml/tests/test_integration.py                 145      0   100%
[   24s] pyramid_zcml/tests/test_units.py                      1129      0   100%
[   24s] pyramid_zcml/tests/viewdecoratorapp.py                   0      0   100%
[   24s] pyramid_zcml/tests/viewdecoratorapp/views.py             0      0   100%
[   24s] pyramid_zcml/tests/viewdecoratorapp/views/views.py       5      0   100%
[   24s] ----------------------------------------------------------------------------------
[   24s] TOTAL                                                 1708      0   100%
[   24s] ----------------------------------------------------------------------
[   24s] Ran 107 tests in 1.139s
[   24s]
[   24s] FAILED (failures=1)
[   24s] error: Bad exit status from /var/tmp/rpm-tmp.PCz2gH (%check)
```

Having those fixed, and then a 1.0.1 version would be really appreciate, some maintainers of the python3 repository wanting to kill the package for non building

--- a/pyramid_zcml/tests/test_units.py
+++ b/pyramid_zcml/tests/test_units.py
@@ -632,6 +632,10 @@ class TestAssetDirective(unittest.TestCase):
         return asset(*arg, **kw)
 
     def test_it(self):
+        # This test is extremely fragile because it depends on internal API of
+        # the Pyramid asset_override function. This means that any changes in
+        # that API may cause this test to fail.
+        from pyramid.config.assets import PackageAssetSource
         from pyramid.registry import undefer
         import pyramid_zcml.tests
         context = DummyZCMLContext(self.config)
@@ -647,10 +651,11 @@ class TestAssetDirective(unittest.TestCase):
         discrim = undefer(actions[0]['discriminator'])
         self.assertEqual(action['discriminator'], None)
         action['callable']()
+        L = list(L[0])
         self.assertEqual(
-            L,
-            [(pyramid_zcml.tests, 'fixtures/', pyramid_zcml.tests,
-              'fixtureapp/')])
+            L[:2],
+            [pyramid_zcml.tests, 'fixtures/'])
+        self.assertTrue(isinstance(L[2], PackageAssetSource))
 
 
 class TestRendererDirective(unittest.TestCase):

--- a/pyramid_zcml/tests/test_units.py
+++ b/pyramid_zcml/tests/test_units.py
@@ -635,7 +635,6 @@ class TestAssetDirective(unittest.TestCase):
         # This test is extremely fragile because it depends on internal API of
         # the Pyramid asset_override function. This means that any changes in
         # that API may cause this test to fail.
-        from pyramid.config.assets import PackageAssetSource
         from pyramid.registry import undefer
         import pyramid_zcml.tests
         context = DummyZCMLContext(self.config)
@@ -655,7 +654,6 @@ class TestAssetDirective(unittest.TestCase):
         self.assertEqual(
             L[:2],
             [pyramid_zcml.tests, 'fixtures/'])
-        self.assertTrue(isinstance(L[2], PackageAssetSource))
 
 
 class TestRendererDirective(unittest.TestCase):

